docs/guides/preparation.md

```diff
@@ -0,0 +1,249 @@
+# Data packaging
```

Comment:
I see the rationale for putting this somewhere else (more so than data formats)...

This is describing _one_ implementation of the standard, and is perhaps more guidance for admins of the RDLS (WB staff/consultants) who are uploading data. So may be more of an _internal_ page, not relevant to lots of people. Happy to consider alternative places, but riskdatalibrary.org may not be most appropriate if it is really an internal doc.

Reply:
Hopefully resolved the inconsistent use of resource and dataset. TBD whether to remove these pages to an internal resource, because while general ideas are broadly applicable to any implementation of a data catalog, there are references here to specific packaging of data in WB Data Catalog implementation.

Reply:
To my mind, we seem to be dealing with at least the following types of guidance:

- How to publish RDLS metadata (not addressed at all by this page, but should definitely form part of the RDLS docs)
- How to structure risk datasets (addressed by this page, but not specific to RDLS)
- How to organise risk datasets in data catalogues (addressed by this page, but not specific to RDLS)
- How to format risk datasets (addressed by `formats.md` and also by this page, but not specific to RDLS)

Reflecting on it, that is an awful lot to cover, so I would encourage us in the first instance to focus on the content that is specific to RDLS (e.g. the unfinished page on [RDL metadata](https://rdl-standard.readthedocs.io/en/docs.mat/guides/rdl-metadata.html#adoption-of-the-metadata-schema)) before addressing those wider issues.

Reply:
Oh, I forgot, there's also:

* How to structure and organise risk datasets for publication in the World Bank Data Catalog (addressed by this page, not RDLS specific, only relevant to WB users)